UnityProject/Assets/Scripts/SpriteHandler/SpriteHandler.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+
+public class SpriteHandler : NetworkBehaviour
+{
+	public SpriteRenderer spriteRenderer;
+
+	public List<SpriteList> spriteList = new List<SpriteList>();
+	[SyncVar(hook = nameof(SyncSprite))] public int spriteIndex;
```

Comment:
I would prefer if spriteIndex was called animationIndex, since you are using it to select which animation is being played, and animationIndex should instead be called spriteIndex, because you are using it select which sprite of the animation to display.